registry/quilt_server/dev_config.py

```diff
@@ -14,8 +14,8 @@
 
 REGISTRY_URL = 'http://localhost:5000'
 
-AWS_ACCESS_KEY_ID = 'fake_id'
-AWS_SECRET_ACCESS_KEY = 'fake_secret'
+AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID', 'fake_id')
```

Comment:
I'm a bit nervous about using the default AWS environment names. Currently, we explicitly override access key and secret with bogus values - but with your change, it will pick up the real ones if they're set using env variables rather than `~/.aws`.

I'd rather call them something like `DEV_ACCESS_KEY_ID`, etc.

Reply:
Is this DEV_ACCESS_KEY or is this an ACCESS key for the service account for the registry?

Reply:
Done.